Makefile

```diff
@@ -247,32 +260,34 @@ push: ## Push Grapl containers to Docker Hub
 
 .PHONY: up
 up: build-services ## Build Grapl services and launch docker-compose up
-	$(WITH_LOCAL_GRAPL_ENV) \
+	$(WITH_LOCAL_GRAPL_ENV)
 	docker-compose -f docker-compose.yml up
 
 .PHONY: up-detach
 up-detach: build-services ## Docker-compose up + detach and return control to tty
 	# Primarily used for bringing up an environment for integration testing.
-	# Usage: `make up-detach PROJECT_NAME=asdf`
-	$(WITH_LOCAL_GRAPL_ENV) \
+	# For use with a project name consider setting COMPOSE_PROJECT_NAME env var
+	# Usage: `make up-detach`
+	$(WITH_LOCAL_GRAPL_ENV)
 	docker-compose \
-		-p $(PROJECT_NAME) \
-		-f docker-compose.yml \
+		--file docker-compose.yml \
```

Comment:
If you explicitly supply `--file` here, any `COMPOSE_FILE` value provided is ignored. That implies that we could harmonize our usage of `TARGET_FILE`/`COMPOSE_FILE` elsewhere (though it would be good to document this intention here at this target; that it's specifically intended to spin up just the local Grapl environment, since that's not particularly evident from the help text comment).

Reply:
I've updated the help text to be more specific that this is intended only for bringing up the local environment only. I considered removing this target since it is now only used by one other target now, but I had seconds thoughts in case anyone has started to use this target directly.

Reply:
I think I'll harmonize the `TARGET_FILE/COMPOSE_FILE` elsewhere and add documentation for that.

Reply:
I agree it'd be nice to harmonize the `TARGET_FILE/COMPOSE_FILE` elsewhere, but it also seems a little tricky to me.

Is there documentation stating the COMPOSE_FILE env var will be ignored when `--file` is provided? I don't see it at:
https://docs.docker.com/compose/reference/overview/#specifying-a-path-to-a-single-compose-file

Reply:
I also went looking for documentation on the interplay between `COMPOSE_FILE` and `--file` but couldn't find it. However, I did some local testing and it looks like it works as I speculated above.

Reply:
I guess we could explicitly unset `COMPOSE_FILE` in `up-detach` to avoid the possibility of that behavior changing. I'd be more comfortable harmonizing with `TEST_FILE` at that point. I'll do that.